x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/tabs/stats.tsx

```diff
@@ -0,0 +1,182 @@
+/*
```

Comment:
I created a `tabs` directory in this PR and https://github.com/elastic/kibana/pull/164628. Let me know if you think that's useful or not.